static/images/pm-amex.svg

```diff
@@ -0,0 +1 @@
2 15"><title>american-express</title><path d="M164.4,11.5V10.2a1.08,1.08,0,0,0-1-1H162v2.3h-1.5V5h3.7a1.69,1.69,0,0,1,1.9,1.4v.5a1.65,1.65,0,0,1-.9,1.5,1.49,1.49,0,0,1,.7,1.5v1.6h-1.5m-2.5-3.8h1.6a.68.68,0,0,0,.8-.4.68.68,0,0,0-.4-.8h-2V7.7m21.3,3.8-2.4-4.4v4.4h-3l-.5-1.4h-2.8l-.5,1.4h-3.2s-2-.3-2-3.1c0-3.4,1.7-3.5,2.4-3.5h1.9V6.4h-1.5a1.24,1.24,0,0,0-1.1,1.4v.5c0,2.2,2.6,1.5,2.7,1.5l1.9-4.9h2.1l2.3,5.9V5h2.1L184,9.3V5h1.5v6.6l-2.3-.1m-8-3h1.5L176,6.6l-.8,1.9m-22.7,3V7l-1.9,4.5h-1.2L147.5,7v4.5h-3l-.5-1.4h-2.8l-.5,1.4H139l2.5-6.6h2.1l2.3,6V5h2.4L150,9.1,151.7,5h2.4v6.6l-1.6-.1m-10.7-3h1.5l-.7-1.9-.8,1.9m13.1,3V5h4.7V6.5h-3.2v1h3.1V9h-3.1v1.1h3.2v1.4h-4.7m11.8.1V5h1.5v6.6h-1.5M170,20V18.7a1,1,0,0,0-1.1-1h-1.6V20h-1.6V13.5h3.9a1.67,1.67,0,0,1,2,1.5v.4a1.65,1.65,0,0,1-.9,1.5,1.76,1.76,0,0,1,.8,1.5V20H170m-2.6-3.8h1.7a.63.63,0,0,0,.4-1.2.3.3,0,0,0-.4,0h-1.7v1.2M159.3,20h-1.4l-1.7-1.9L154.5,20h-6V13.5h5.9l1.8,2.1,1.9-2.1h5.1a1.73,1.73,0,0,1,2,1.5v.4c0,1.9-.7,2.5-2.8,2.5h-1.6V20h-1.5m-2-3.2,2,2.3V14.6l-2,2.2m-7.2,1.8h3.4l1.6-1.8L153.5,15h-3.4v1h3.3v1.5h-3.3v1.1m10.8-2.4h1.7A.63.63,0,0,0,163,15a.3.3,0,0,0-.4,0h-1.7v1.2M181,20h-2.9V18.5h2.5s.9.1.9-.5-1.4-.5-1.4-.5a1.93,1.93,0,0,1-2.2-1.6v-.3a1.88,1.88,0,0,1,1.7-2H183V15h-2.5s-.9-.2-.9.5c0,.5,1.2.5,1.2.5s2.5-.2,2.5,1.8a2.1,2.1,0,0,1-1.8,2.3h-.4L181,20m-8.6,0V13.5h5V15H174v1h3.3v1.5H174v1.1h3.4V20h-5m14.2,0h-2.9V18.5h2.5s.9.1.9-.5-1.4-.5-1.4-.5a1.93,1.93,0,0,1-2.2-1.7v-.3a1.88,1.88,0,0,1,1.7-2h3.4V15h-2.5s-.9-.2-.9.5c0,.5,1.2.5,1.2.5s2.5-.2,2.5,1.8a2.1,2.1,0,0,1-1.8,2.3h-.4c0-.1,0-.1-.1-.1m46.7,4h-4.1A11.42,11.42,0,0,1,218,12.6,11.17,11.17,0,0,1,228.7,1h4.6A11.61,11.61,0,0,1,245,12.5v.1A11.72,11.72,0,0,1,233.3,24ZM229.5,3a9.5,9.5,0,1,0,9.5,9.5A9.56,9.56,0,0,0,229.5,3ZM231,18.9V6.2a6.54,6.54,0,0,1,0,12.7Zm-3,0a6.56,6.56,0,0,1,0-12.7ZM277.5,8H275v8.9h2.5a4.45,4.45,0,0,0,3.1-1,4.58,4.58,0,0,0,1.6-3.4A4.35,4.35,0,0,0,278,8Zm2,6.7a3.17,3.17,0,0,1-2.3.7h-.5V9.5h.5a3.17,3.17,0,0,1,2.3.7,3.1,3.1,0,0,1,.9,2.2A3.17,3.17,0,0,1,279.5,14.7Zm3.4,2.2h1.7V8h-1.7Zm5.8-5.5c-1-.4-1.3-.6-1.3-1.1a1.11,1.11,0,0,1,1.2-1,2.1,2.1,0,0,1,1.4.7l.9-1.2a4,4,0,0,0-2.5-1,2.56,2.56,0,0,0-2.7,2.4v.1c0,1.2.5,1.9,2.1,2.4a6.12,6.12,0,0,1,1.2.5,1.05,1.05,0,0,1,.5.9,1.31,1.31,0,0,1-1.2,1.3h-.1a2,2,0,0,1-1.8-1.2l-1.1,1.1a3.4,3.4,0,0,0,3,1.7,2.82,2.82,0,0,0,3-2.7V14C291.2,12.8,290.7,12.2,288.7,11.4Zm3.1,1.1a4.57,4.57,0,0,0,4.5,4.6,5.45,5.45,0,0,0,2.2-.5v-2a2.61,2.61,0,0,1-2,1,2.82,2.82,0,0,1-2.9-2.8v-.2a3,3,0,0,1,2.8-3.1,2.81,2.81,0,0,1,2.1,1v-2a3.84,3.84,0,0,0-2.1-.5A4.33,4.33,0,0,0,291.8,12.5ZM312.1,14l-2.3-6H308l3.7,9.1h.9L316.3,8h-1.8l-2.4,6Zm5,2.9h4.8V15.4h-3.1V13h3V11.5h-3v-2h3.1V8h-4.8Zm11.5-6.3c0-1.7-1.1-2.6-3.1-2.6H323v8.9h1.7V13.3h.2l2.3,3.6h2.1l-2.7-3.7a2.43,2.43,0,0,0,2-2.6Zm-3.4,1.5h-.5V9.4h.5c1.1,0,1.6.5,1.6,1.3S326.3,12.1,325.2,12.1Z" style="fill:#aab7c4"/><path d="M303.7,7.9a4.59,4.59,0,0,0-4.6,4.6,4.74,4.74,0,0,0,4.6,4.7,4.61,4.61,0,0,0,4.7-4.6A4.76,4.76,0,0,0,303.7,7.9Z" style="fill:#aab7c4;opacity:0.5;isolation:isolate"/></svg>
```

Comment:
Can you run all the SVGs through [svgomg](https://jakearchibald.github.io/svgomg/)